navigation/MainTabNavigator.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import { Platform, View } from 'react-native';
 import {
   createStackNavigator,
   createMaterialTopTabNavigator,
 } from 'react-navigation';
 import { Entypo, Feather, MaterialIcons } from '@expo/vector-icons';
+import { BlurView } from 'expo-blur';
```

Comment:
FINALLY!!! I've been trying to update react-native on my own so I could use this. Yay!!!